vllm_ascend/distributed/kv_transfer/llmdatadist_pipe.py

```diff
@@ -0,0 +1,218 @@
+#
+# Copyright (c) 2025 Huawei Technologies Co., Ltd. All Rights Reserved.
+# This file is a part of the vllm-ascend project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import Optional
+import zmq # type: ignore
+import msgpack # type: ignore
+import time
+import threading
+
+import torch
+import torch_npu
+import torchair  # type: ignore
+
+from vllm.distributed.kv_transfer.kv_pipe.base import KVPipeBase
+from vllm.logger import init_logger
+from vllm.utils import get_ip
+
+import vllm_ascend.envs as envs
+
+import llm_datadist  # type: ignore
+
+logger = init_logger(__name__)
+
+
+NPU_DTYPE_TO_TORCH_DTYPE = {
```

Comment:
The datatype conversion has been used in both pipe and buffer files. Can we have a util file to cover this to avoid duplicated code?